docs/docs/adrs/adr-005-cryptographic-equivocation-verification.md

```diff
@@ -0,0 +1,36 @@
+---
+sidebar_position: 4
+title: Cryptographic verification of equivocation evidence
+---
+# ADR 005: Cryptographic verification of equivocation evidence
+
+## Changelog
+* 5/1/2023: First draft
+
+## Status
+
+Proposed
+
+## Context
+
+Currently, we use a governance proposal to slash validators for equivocation (double signing and light client attacks). This has the downside that it takes 2 weeks for the proposal to be approved, effectively reducing the unbonding period in some respects. This does not lead to any pressing real-world security concerns, but since it involves the basis of proof of stake, it would be good to get consumer chain slashing back to parity as soon as possible.
```

Comment:
> effectively reducing the unbonding period in some respects

I think this reduces the trusting period and not the unbonding period. Given a 21 days unbonding period and a 14 days trusting period, for standalone chains, a client would expire if it's not updated within 14 days (i.e., client updates are no longer trusted). Now, due to the 2 weeks proposal for consumer chains, the trusting period must be smaller than 21 day minus 2 weeks, so that by the time the proposal passes, the misbehaving stake is still locked.